nativescript-angular/testing/setup.js

```diff
@@ -0,0 +1,19 @@
+require("reflect-metadata");
+require("nativescript-angular/zone-js/dist/zone-nativescript");
+require("zone.js/dist/long-stack-trace-zone");
```

Comment:
NativeScript bundles its own version of zone.js, and requiring `zone.js` from the npm looks fishy. You could easily end up with two separate zone implementations and that will likely break things. I suggest we use `nativescript-angular/zone-js` everywhere.